1_fetch.R

```diff
@@ -0,0 +1,41 @@
+source("1_fetch/src/find_nwis_sites.R")
+source("1_fetch/src/download_DO_data.R")
+
+p1_targets_list <- list(
+  # Load harmonized WQP data product
+  tar_target(
+    p1_wqp_data,
+    readRDS(file = "1_fetch/in/DRB.WQdata.rds")),
```

Comment:
Sure, good question! `DRB.WQdata.rds` is a copy of the multiscale surface water quality data for the DRB (Shoda et al. 2019, data release [here](https://www.sciencebase.gov/catalog/item/5d6fa631e4b0c4f70cf92e9e)). The Shoda et al. dataset represents a QA/QC'ed version of the data available on the Water Quality Portal through early 2019. Ideally, I would pull this in programmatically with `sbtools`, but added this file to an `/in/` directory to get around issues with ScienceBase going down, at least temporarily.

Reply:
ah gotcha. thanks for that explanation. 27 MB is getting up into the range where I think we should think carefully if we want to "host" this in our github repo. I don't love the idea of totally relying on SB, but I since we have to rely on SB for other parts of this project that will have much bigger files (I.e., the met drivers that will be ~1 GB), I think it's best if we rely on it here too.

Reply:
I agree in terms of storage size and it's a good point about relying on the met drivers that are on SB. I'll go ahead and bring this in with ScienceBase. The data shouldn't need frequent rebuilds and I can try to think about other ways to build in some fault tolerance for when SB goes down.